row_047_app.py

```
@@ -0,0 +1,128 @@
+
+
+@app.route("/api/clean", methods=["POST"])
+def clean():
+    data = {"success": False}
+    minutes = flask.request.form.get("minutes", type=str, default=30)
+    if flask.request.method == "POST":
+        cmd_in = "find ./static/input/ -type f -mmin +" + minutes + " -exec rm {} \;"
+        cmd_out = "find ./static/output/ -type f -mmin +" + minutes + " -exec rm {} \;"
+        cmd_bg = "find ./static/bg/ -type f -mmin +" + minutes + " -exec rm {} \;"
+        cmd_merge = "find ./static/merge/ -type f -mmin +" + minutes + " -exec rm {} \;"
+        os.system(cmd_in)
```

Ruff (0.13.2)

Starting a process with a shell, possible injection detected

(S605)